packages/accordion/demo/stories/AccordionStory.tsx

```diff
@@ -28,34 +30,34 @@ const Component = ({
   getPanelProps
 }: IComponentProps) => (
   <div style={{ width: 300 }}>
-    {sections.map((section, index) => {
-      const disabled = disabledSections.indexOf(index) !== -1;
-      const hidden = expandedSections.indexOf(index) === -1;
+    {sections!.map(value => {
+      const disabled = disabledSections.indexOf(value) !== -1;
+      const hidden = expandedSections.indexOf(value) === -1;
 
       return (
-        <div key={section}>
-          <div {...getHeaderProps({ ariaLevel: 2 })}>
+        <div key={value}>
+          <div {...getHeaderProps({ 'aria-level': 2 })}>
             <button
               {...getTriggerProps({
-                index,
+                value,
                 role: null,
                 tabIndex: null,
                 disabled
               })}
               className="text-left w-full"
               type="button"
             >
-              {`Trigger ${index + 1}`}
+              {`Trigger ${value + 1}`}
```

Comment:
```suggestion
              {`Trigger ${index + 1}`}
```

Please revert .. we shouldn't assume what the user will provide as control values.